libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/daemon/MessageBusListenerActor.scala

```diff
@@ -32,9 +36,11 @@ class MessageBusListenerActor[M](source: Source[M, NotUsed])(implicit messageLik
         log.warning("Listener already subscribed. Ignoring Subscribe message")
       case Failure(ex) =>
         log.error(ex, "Source/Listener died, subscribing again")
+        monitor.onError(ex)
         trySubscribeDelayed()
```

Comment:
Had a look, I use it in Auth+, Device Gateway. So should be possible. Let's have a look later